@uportal/portlet-card/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "portlet-card",
```

Comment:
It can be helpful to prefix names like `portlet-card` with a namespace to avoid npm naming conflicts.
E.G. `@uportal/portlet-card`